docs/sources/plugins/developing/plugin.json.md

```diff
@@ -12,17 +12,28 @@ weight = 8
-| info.version | project version of this commit. Must be semver |
-| dependencies.grafanaVersion | Required grafana backend version for this plugin |
-| dependencies.plugins | required plugins for this plugin. |
+| Property                    | Description                                                                                                                   |
+| --------------------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| id                          | Unique name of the plugin. See [naming conventions described in styleguide]({{< relref "code-styleguide.md" >}})              |
+| type                        | One of `app`, `datasource`, `panel`                                                                                           |
+| name                        | Human-readable name of the plugin                                                                                             |
+| dependencies.grafanaVersion | Required Grafana version for this plugin                                                                                      |
+| dependencies.plugins        | An array of required plugins on which this plugin depends                                                                     |
+| info.author.name            | Author's name                                                                                                                 |
+| info.author.url             | Link to author's website                                                                                                      |
+| info.description            | Description of plugin. Used for search on grafana.com                                                                         |
+| info.keywords               | Array of plugin keywords. Used for search on grafana.com                                                                      |
+| info.links                  | An array of link objects to be displayed on this plugin's project page in the form `{name: 'foo', url: 'http://example.com'}` |
+| info.logos.small            | Link to the "small" version of the plugin logo, which must be an SVG image. "Large" and "small" logos can be the same image. |
+| info.logos.large            | Link to the "large" version of the plugin logo, which must be an SVG image. "Large" and "small" logos can be the same image. |
+| info.screenshots            | An array of screenshot objects in the form `{name: 'bar', path: 'img/screenshot.png'}`                                        |
+| info.updated                | Date when this plugin was built. Use `%TODAY%` for Grafana to autopopulate this value.                                        |
+| info.version                | Project version of this commit. Use `%VERSION%` for Grafana to autopopulate this value.                                      |
+
+# Plugin.json Example
```

Comment:
Shouldn't this be a level 2 heading? I.e.,
```suggestion
## Plugin.json Example
```

Reply:
Yes it should, thank you